lib/manageiq/loggers/base.rb

```diff
@@ -91,7 +103,16 @@ def self.log_hashes(logger, h, options = {})
         logger.send(level, "\n#{values}")
       end
 
+      def log_hashes_filter
+        @log_hashes_filter || self.class.log_hashes_filter
+      end
+
+      def log_hashes_filter=(filters)
+        @log_hashes_filter ||= self.class.send(:clean_log_hashes_filter, filters)
```

Comment:
There is no such thing as a `protected_class_method`, which is what I want here, so I settled for a send ☹️)

Reply:
How is this different from the code below?

```suggestion
        @log_hashes_filter ||= self.class.clean_log_hashes_filter(filters)
```

Reply:
Because I've declared `clean_log_hashes_filter` a `private_class_method` since I don't want it in the public interface. What I want is `protected_class_method`, but that doesn't exist.